doc/sphinx-guides/source/admin/make-data-count.rst

```diff
@@ -0,0 +1,161 @@
+
+.. contents:: Contents:
+	:local:
+
+Introduction
+------------
+
+`Make Data Count`_ is part of a broader Research Data Alliance (RDA) `Data Usage Metrics Working Group`_ which helped to produce a specification called the `COUNTER Code of Practice for Research Data`_ (`PDF`_, `HTML`_) that Dataverse makes every effort to comply with. The Code of Practice (CoP) is built on top of existing standards such as COUNTER and SUSHI that come out of the article publishing world.  The Make Data Count project has emphasized that they would like feedback on the code of practice. You can keep up to date on the Make Data Count project by subscribing to their `newsletter`_.
+
+Architecture
+------------
+
+Dataverse installations who would like support for Make Data Count must install `Counter Processor`_, a Python project created by California Digital Library (CDL) which is part of the Make Data Count project and which runs the software in production as part of their `DASH`_ data sharing platform.
+
+.. _Counter Processor: https://github.com/CDLUC3/counter-processor
+.. _DASH: https://cdluc3.github.io/dash/
+
+The diagram below shows how Counter Processor interacts with Dataverse and the DataCite hub, once configured. Installations of Dataverse using Handles rather than DOIs should note the limitations in the next section of this page.
+
+|makedatacount_components|
+
+The most important takeaways from the diagram are:
+
+- Once enabled, Dataverse will log activity (views and downloads) to a specialized date-stamped file.
+- You should run Counter Processor once a day to create reports in SUSHI (JSON) format that are saved to disk for Dataverse to process and that are sent to the DataCite hub.
+- You should set up a cron job to have Dataverse process the daily SUSHI reports, updating the Dataverse database with the latest metrics.
+- You should set up a cron job to have Dataverse pull the latest list of citations for each dataset on a periodic basis, perhaps weekly or daily. These citations come from Crossref via the DataCite hub.
+- APIs are available in Dataverse to retrieve Make Data Count metrics: views, downloads, and citations.
+
+
+Limitations for Dataverse Installations Using Handles Rather Than DOIs
+----------------------------------------------------------------------
+
+Data repositories using Handles and other identifiers are not supported by Make Data Count but in the notes_ following a July 2018 webinar, you can see the Make Data Count project's response on this topic. In short, the DataCite hub does not want to receive reports for non-DOI datasets. Additionally, citations are only available from the DataCite hub for datasets that have DOIs. The Dataverse usage logging and Counter Processor tool can still be used to track other identifier and store the metrics in Dataverse.
```

Comment:
If an installation has other identifiers besides DataCite, they can still generate logs and process those logs with Counter Processor to create json that details usage on a dataset level. Dataverse can ingest this locally generated json. They are NOT able to use Counter Processor to send the logs to DataCite/Make-Data-Count as DataCite will outright reject the whole json file even if there is only one Handle PID. Sending the json to Make-Data-Count was a requirement for the Harvard Dataverse installation which is why we went with converting the last of our Handles to DataCite, but other installations can still get detailed usage information out of this work.

As Make-Data-Count advances hopefully their APIs will become more verbose and we won't have to worry about a single PID breaking the flow. Or heck, maybe they'll become PID agnostic. We could also at some point fork Counter-Processor to better support these mixed PID installations (e.g. create two copies of the JSON, one with only DataCite PIDs for sending off to Make Data count), but for the first pass this seemed like the easiest route.

I'll go back over the documentation and see how this can be more clear, as its definitely confusing.

Reply:
FWIW, this is the data flow:
- Dataverse generates separate logs for page views / downloads
- Counter Processor consumes these logs, weeding out repeated clicks, geolocating IP addresses, etc. This aggregated data is placed in a json file.
   - If enabled, Counter Processor sends this json file to Make Data Count.
- Dataverse consumes the locally generated json file and populates a table with the information.